src/views/IDUploader/index.js

```diff
@@ -26,6 +25,9 @@ import IdCardDefault from './image-default.png';
 import IdCardGreen from './image-green.png';
 import IdCardTop from './image-top.png';
 
+// FIXME checkout https://mui.com/components/use-media-query/#migrating-from-withwidth
+const withWidth = () => (WrappedComponent) => (props) => <WrappedComponent {...props} width="xs" />;
+
```

Comment:
Since the point of this pull request is to upgrade to MUI v5, wouldn't it be best to replace this hook rather than re-creating the logic ourselves?

Reply:
Probably, yes. This was created by a codemod, I did not make this change myself.

However, given that this entire component needs to be re-written (and the styling deeply reconsidered), I decided that it wasn't worth my time to change that right now.

Reply:
Alright